meerwatch_tools.py

```diff
@@ -120,8 +134,13 @@ def plot_toas_fromarr(toas, pid="unk", mjd=None, fs=14, out_file="toas.png", out
     # use semi-fixed color normalisation
     f_min = cfrq-bw/2.0
     f_max = cfrq+bw/2.0
+    norm = Normalize(vmin=f_min, vmax=f_max)
```

Comment:
I'm not sure I understand the significance of these changes. If they produce the same plots as before I'm okay with them, but I might need you to walk me through them.

Reply:
It seems the more up-to-date version of `matplotlib` on NT wasn't happy with the previous method so it gave errors. These changes make the same plots ad the previous implemtation